src/main/java/teammates/common/datatransfer/InstructorPrivileges.java

```diff
@@ -434,6 +441,22 @@ private boolean isAllowedInSessionLevel(String sectionName, String sessionName,
         }
     }
     
+    private boolean isAllowedInSessionLevelAnySection(String sessionName, String privilegeName) {
+        for (String sectionName : this.sessionLevel.keySet()) {
+            if (this.sessionLevel.get(sectionName).get(sessionName).containsKey(privilegeName) &&
```

Comment:
Oh i get what you mean now. No need to check for it as we are iterating through the key set. It must exist.